### PR TITLE
Fix firmware-5.bin link in wget command

### PR DESCRIPTION
**Pull request by RussianNeuroMancer**
Fix firmware-5.bin link to make wget download firmware-5.bin instead of html page

### CODE REVIEW

File: misc/lenovo-yoga-c630/wifi/README.md [hunk 1]
```diff
@@ -137,4 +137,4 @@ $  sudo systemctrl enable qrtr-ns
 $  sudo systemctrl enable pd-mapper
 $  sudo systemctrl enable tqftpserv
 $  sudo systemctrl enable rmtfs
-```
\ No newline at end of file
+```
```
Comment (lag-linaro) [inline misc/lenovo-yoga-c630/wifi/README.md]: ?

**Comment by lag-linaro** [comment]
Is this still relevant?

I'm a little confused by it.  Why are you submitting 5 patches which I authored?

What is the relevance of the final line in your patch?

**Comment by RussianNeuroMancer** [comment]
Sorry for late answer. I didn't lookup into e-mail inbox during vacation.

> Is this still relevant?

If [this document](https://github.com/aarch64-laptops/build/blob/master/misc/lenovo-yoga-c630/wifi/README.md) is still relevant, then - yes. Currently [this command](https://github.com/aarch64-laptops/build/blob/master/misc/lenovo-yoga-c630/wifi/README.md) still download html page and save it as "firmware-5.bin" which obviously break WiFi driver. #27 probably is also still relevant, unless newer version of rmtfs can build without qmic.

> Why are you submitting 5 patches which I authored?

As this pull request was submitted 11 month ago I honestly don't remember why this patches is there. Probably I edited README.md in wrong repository and submitted pull request without noticing that there is four unrelated changes.

> What is the relevance of the final line in your patch?

As above - I don't remember. Maybe bug in GitHub editor, I don't know.

Probably it will be easier if you apply both changes (corrected link to firmware-5.bin and qmic changes from #27) yourself.

**Comment by lag-linaro** [comment]
I fixed the `wget` command.

**Comment by lag-linaro** [comment]
Please could you resubmit #27

**Pull request closed without merging**